specutils/analysis/flux.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division
+
+import numpy as np
+from astropy.units.quantity import Quantity
+from .utils import computation_wrapper
+
+
+__all__ = ['line_flux', 'equivalent_width']
+
+
+def line_flux(spectrum, region=None):
+    """
+    Computes the line flux over a spectrum. Applies a region if given.
+
+    Parameters
+    ----------
+    spectrum : Spectrum1D
+        The spectrum object over which the line flux will be calculated.
+
+    Returns
+    -------
+    flux : `~astropy.units.Quantity`
+        line flux result
+    """
+    return computation_wrapper(_compute_line_flux, spectrum, region)
+
+
+def equivalent_width(spectrum, continuum=1, region=None):
+    """
+    Does a naive equivalent width measures on the spectrum object.
```

Comment:
Instead maybe "The equivalent width of a region of the spectrum".

EQW's don't *really* make sense if there's no sub-region. Nominally it's a well-defined object but no one really uses it that way.